plugins/identity/app/controllers/identity/domains/technical_users_controller.rb

```diff
@@ -4,17 +4,23 @@ module Identity
   module Domains
     class TechnicalUsersController < ::DashboardController
 
+      before_filter do
+        @policy_default_params ||= {}
+        @policy_default_params[:user] = { domain_id: @scoped_domain_id }
```

Comment:
no, current_user doesn't have the attribute scoped_domain_id
But there is domain_id and user_domain_id. So you could change the rule in policy